script/Ciel/Day 5/05_01_CIEL05_1E.txt

```diff
@@ -572,6 +572,7 @@ Her long-handled weapon in hand, the nun tramples the Dead as though dancing.
 -- Page 242, Offset 11501.
 -- 弾けた笑顔で、目を輝かせながら、修道服のスカートもバンバンひるがえして、時には投げキッスまで<披露|サービス>する、まさに年齢考えろ的な、それは決め決めのアクションだった。
 // Not able to find any refs for 決め決め but I guess deliberate makes sense?
+// group pr check: can be improved
```

Comment:
was this the right line? I don't see anything wrong

Reply:
I think so. Let's just leave it as is if there isnt anything to improve